editoast/openapi_legacy.yaml

```diff
@@ -3892,6 +3892,10 @@ components:
           type: integer
         path_length:
           type: integer
+        update_required:
```

Comment:
It is indeed, but it will be done in a next PR that is the next step of this feature to not break the front. Currently, some api endpoints are still used by the front and if update_required is required atm, some issues will happen in the front ! (seen with @SharglutDev)